common/src/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -408,6 +408,11 @@ public final class LC {
     @Supported
     public static final KnownKey ldap_starttls_required = KnownKey.newKey(true);
 
+    @Reloadable
+    public static final KnownKey clamav_socket_timeout = KnownKey.newKey(2000);
```

Comment:
the timeout & size used as default, why are these values selected?

Reply:
I did not get your question

Reply:
just trying to understand how these were selected as default values

Reply:
why @Reloadable?

Reply:
OK.got it. No specific reason but tried to keep it as ideal as possible. This can vary from setup to setup, for this reason, we introduced LC attributes so they can be controlled.

Reply:
I think Redoable is OK if server restart is not required.
`If it's reloadable, changes are in effect
     * after LC reload. Otherwise, changes are in effect after server restart.`